schedule/qam/common/qam-yast_self_update_libyui-rest.yaml

```diff
@@ -0,0 +1,37 @@
+---
+name: qam-yast_self_update_libyui-rest
```

Comment:
new schedule should be under `/schedule/yast/
we don't have a clear idea about the structure, there is some folder for SLE-15-SP3, chose some, will refactor, but at least it should be in our folder.